src/controllers/auth/handlers/login.handler.ts

```diff
@@ -0,0 +1,20 @@
+import { Unauthorized, createAccessToken } from "@panenco/papi"
+import { LoginBody } from "../../../contracts/login.body"
+import { RequestContext } from "@mikro-orm/core"
+import { User } from "../../../entities/user.entity"
+import { MyToken } from "../../../contracts/admin.requirements/token.type"
+
+export const login = async (body: LoginBody) => {
+    const em = RequestContext.getEntityManager();
+    const user = await em.findOneOrFail(User, {email: body.email})
+    if (user.attempts >= 3) {
+        throw new Unauthorized('TooManyTries', 'Too many tries');
+    }
+    if (body.password !== user.password) {
+        user.assign({attempts: user.attempts + 1});
+        await em.flush();
+        throw new Unauthorized('IncorrectPassword', 'Password is incorrect');
+    }
+    const token = await createAccessToken('jwtSecretFromConfigHere' , 600, {userId: user.id, role: user.role} as MyToken);
```

Comment:
ideally should be extracted to an environment variable and some config object so we can re-use it

```ts
// config.ts

export const config = {
  jwtSecret: process.env.JWT_SECRET,
};
```